packages/pages/src/components/breadcrumbs/yextBreadcrumbs.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Breadcrumbs from "./breadcrumbs.js";
+import { BreadcrumbsProps } from "./types.js";
+import { defaultTransformer } from "./methods.js";
+
+/*
+ * Representation of how breadcrumbs are received from Yext Streams
+ */
+export interface StreamsBreadcrumbs {
+  dm_directoryParents: Array<StreamsBreadcrumb>;
+}
+
+interface StreamsBreadcrumb {
+  slug: string;
+  name: string;
+}
+
+export interface BreadcrumbsPropsStreams extends BreadcrumbsProps {
+  streamsBreadcrumbs: StreamsBreadcrumbs;
+}
+
+/*
+ * Wrapper for the Breadcrumbs component which automatically processes Yext Streams data
+ */
+const YextBreadcrumbs = (props: BreadcrumbsPropsStreams) => {
```

Comment:
What's the purpose of this instead of just using the Breadcrumbs directly?

Reply:
This was supposed to be an adapter template to unmarshall whatever the Streams breadcrumbs data format was into the generic `[string, string]` format